addons/account/demo/account_demo.py

```diff
@@ -29,6 +28,37 @@ def _get_demo_data(self, company=False):
             'mail.activity': self._get_demo_data_mail_activity(company),
         }
 
+    def _get_demo_exception_product_template_xml_ids(self):
+        """ Return demo product template xml ids to not put taxes on"""
+        return []
+
+    def _get_demo_exception_product_variant_xml_ids(self):
+        """ Return demo product variant xml ids to not put taxes on"""
+        return ['product.office_combo', 'loyalty.ewallet_product_50', 'loyalty.gift_card_product_50']
```

Comment:
Since `loyalty` depends on `account`, shouldn't we set the exception over there?

Reply:
@william-andre It should work without setting the exception at all.  I will check.